funnel/models/event.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+from . import db, BaseMixin
+from .space import ProposalSpace
+from .user import User
+import random
+import uuid
+from sqlalchemy.ext.associationproxy import association_proxy
+from datetime import datetime
+
+__all__ = ['Event', 'TicketType', 'Participant', 'Attendee', 'SyncTicket']
+
+PRINTABLE_ASCII = map(chr, range(32, 127))
+
+
+def make_key():
+    key = str(uuid.uuid4()).replace('-', '')
+    return ''.join(random.sample(key, len(key)))
+
+
+def make_public_key():
+    return make_key()[:8]
+
+
+def make_private_key():
+    return make_key()[:8]
+
+
+event_ticket_type = db.Table('event_ticket_type', db.Model.metadata,
```

Comment:
What is this table? Explara ticket categories? Then it needs a column with a textual definition of the ticket type.